packages/suite/src/views/settings/device/index.tsx

```diff
@@ -204,8 +203,8 @@ const Settings = () => {
                                 }}
                                 isDisabled={
```

Comment:
Just a nitpick - I'd maybe put these kind of conditions into a helper fn - it might become more readable

Reply:
Probably better to leave it for some refactoring task. Conditions like this are all over the project.